main/core.js

```diff
@@ -43,7 +44,9 @@ async function setup (ctx) {
     }
   }
   await maybeMigrateFiles()
-  await start(ctx)
+  while (true) {
+    await start(ctx)
+  }
```

Comment:
I have a proposal to consider:

Let's keep the current `setup` semantics that's consistent with other Station Desktop backend components.

Change the `setup` function to return `run` function, change the code calling `setup` to collect all `run` functions and execute them as the last step.

```diff
- await start(ctx)
+ return run
```

Later in `index.js`:

```diff
- await core.setup(ctx)
+ const runCore = await core.setup(ctx)
+ await runCore(ctx)
```

This design makes it easy to add more long-running tasks in the future.

OTOH, maybe we don't need it yet and it's enough to rename `setup` to `run` 🤷🏻 

```diff
- await core.setup(ctx)
+ await core.run(ctx)
```

Reply:
After https://github.com/filecoin-station/desktop/pull/1549/commits/896b93778b11b0b7c4a056ec0d69a4be718b2e64, would you still propose this?

Reply:
I love https://github.com/filecoin-station/desktop/commit/896b93778b11b0b7c4a056ec0d69a4be718b2e64 😍